tests/test_ctx_options.cpp

```diff
@@ -283,6 +283,14 @@ void test_ctx_option_blocky ()
     test_context_socket_close (router);
 }
 
+void test_ctx_option_invalid ()
+{
+    TEST_ASSERT_EQUAL_INT (-1, zmq_ctx_set (get_test_context (), -1, 0));
```

Comment:
Just for the future: You can simply write:
```
     TEST_ASSERT_FAILURE_ERRNO (EINVAL, zmq_ctx_set (get_test_context (), -1, 0));
```
instead of the two separate assertions.